lib/ae_mdw_web/controllers/aex9_controller.ex

```diff
@@ -177,6 +181,16 @@ defmodule AeMdwWeb.Aex9Controller do
   #
   # Private functions
   #
+  defp by_contract_reply(conn, contract_id) do
+    entry =
+      case Contract.aex9_search_contract_by_id(contract_id) do
+        {:ok, rev_aex9_key} -> Format.to_map(rev_aex9_key, Model.RevAex9Contract)
+        :not_found -> %{}
```

Comment:
Wouldn't we want to return 404 here?

Reply:
We could make it different on v2 even though it's not a rest api. 
For now it would be better like this because the other kinds of aex9 searches return 200 with empty result:
https://mainnet.aeternity.io/mdw/aex9/by_name?exact=AAAB

Reply:
You're right, for backwards compat we should keep this behavior